doc/content/getting-started/installation/configuration/_index.md

```diff
@@ -44,11 +52,13 @@ Download the example `ttn-lw-stack-docker.yml` for {{% tts %}} Open Source <a hr
 {{< /tabs/tab >}}
 {{< /tabs/container >}}
 
-{{< note >}} These example configuration files contain all of the configuration settings you need to run {{% tts %}} for development - just update the files with your server address. {{</ note >}}
+{{< note >}} These example configuration files contain all of the configuration settings you need to run {{% tts %}} for development. Be sure to update `ttn-lw-stack-docker.yml` with your server address, generate keys in `http.cookie` and set `console.oauth.client-secret`.
+
+For an extended explanation of the configuration settings, keep reading. {{</ note >}}
 
 Settings in `docker-compose.yml` and `ttn-lw-stack-docker.yml` files are explained in detail in [Docker Configuration](#docker-configuration) and [The Things Stack Configuration](#the-things-stack-configuration) sections. Further, we provide tips for running {{% tts %}} in production.
 
```

Comment:
These references don't lead me to the right place, but it could be something on my side, please just double check this